### PR TITLE
[VS2013] Project structure improvement with nested projects and missi…

### PR DESCRIPTION
**Pull request by AchimTuran**
This PR will improve the VS2013 project structure with nested projects to improve the overview. It also adds missing header files, which are used by the binary add-ons.

TODO: add a screenshot of the nested projects.

@FernetMenta: is there anyone else who should I ping?

### CODE REVIEW

**Comment by Paxxi** [comment]
Most likely me and @Montellese for review. Probably won't have enough time until sunday

**Comment by afedchin** [comment]
As I know x64 isn't complete yet. Why did you add x64 configs?

**Comment by afedchin** [comment]
ah. VS did that. so also need to drop (DDL) and Profile configs.

File: project/VS2010Express/XBMC for Windows.sln [hunk 1]
```diff
@@ -68,178 +68,566 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Effects11", "..\..\lib\win3
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "libKODI_adsp", "..\..\lib\addons\library.kodi.adsp\project\VS2010Express\libKODI_adsp.vcxproj", "{44F93C4D-85DD-4452-99BB-F1D196174024}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "libKODI", "libKODI", "{A383FE5D-2EEE-4833-83F7-B4E3EEAD4E27}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ImageLib", "ImageLib", "{78BFF850-3CA7-4CBE-9BEC-190F782263F0}"
```
Comment (afedchin) [inline project/VS2010Express/XBMC for Windows.sln]: I think no need a defferent folder for Image lib. Keep it in `libs` folder.

**Comment by AchimTuran** [comment]
As I know x64 isn't complete yet. Why did you add x64 configs? It was VS2013 here, but I can remove them.

File: project/VS2010Express/XBMC for Windows.sln (hunk 1 above)
Comment (AchimTuran) [inline project/VS2010Express/XBMC for Windows.sln]: Yeah no problem.

**Comment by AchimTuran** [comment]
If you want I can add a screenshot, but if you know the feature from VS it is not necessary.

**Comment by afedchin** [comment]
> It was VS2013 here, but I can remove them.
please do that.

**Comment by AchimTuran** [comment]
I closed this PR, because it is obsolete with our new CMake based system.

**Pull request closed without merging**